.gitignore

```diff
@@ -3,3 +3,6 @@ typesafe_conductr_cli.egg-info
 __pycache__
 deb_dist/*
 !deb_dist/Dockerfile
+.idea
+*.iml
```

Comment:
Yeah, and therefore these patterns weren't present for me which led to adding them to Git accidentally.